R/customVarInfo.R

```diff
@@ -20,17 +20,59 @@
 
 
 
-# TODO: clean up and create sensible varInfos!!!!!
+
+#' @export
+getCustomInfo <- function(v, info, default = NULL, onError = NULL) {
```

Comment:
Is it necessary to export this function?

Reply:
I think it would be nice for users to have the possibility to add their own `varInfo`, in case they are using some more complex custom classes etc., so currently I exported most of the functions used to manipulate `varInfo`s.